account_payment_return_import/tests/test_import_base.py

```diff
@@ -40,26 +40,23 @@ def setUpClass(cls):
         cls.reason = cls.env["payment.return.reason"].create(
             {"code": "RTEST", "name": "Reason Test"}
         )
-        cls.account_type = cls.env["account.account.type"].create(
-            {"name": "Test", "type": "receivable", "internal_group": "asset"}
-        )
+        # cls.account_type = cls.env["account.account.type"].create(
```

Comment:
Why did you comment this code?, if it is not used it is better to remove it.
you should avoid to commit things like this